### ISSUE TITLE
TypeError when feature flag rollout percentage is None

### ISSUE THREAD

**Issue by samwinslow** [PostHog/posthog#5556]
## Bug description

Please see the Sentry error report here: https://sentry.io/organizations/posthog/issues/2569994933/?project=1899813&query=is%3Aunresolved&sort=new&statsPeriod=14d

Simple feature flags that are defined without a rollout percentage seem to trip up the decision logic.

## Expected behavior

Feature flags without rollout percentages are assumed to be 100% for all math calculations

## How to reproduce

(Sorry this is vague — still gathering more context)

1. Create a flag without a rollout percentage
2. Access it

## Environment

- [ ] PostHog Cloud
- [ ] self-hosted PostHog, version/commit: _please provide_

**Comment by samwinslow** [comment]
I'm keen to take this on — or at least see if it's a simple fix. It will impede our progress toward multi-variant feature flags (#5440) if left unresolved.

**Comment by samwinslow** [comment]
Huh, looks like https://github.com/PostHog/posthog-python/pull/30 should have fixed this, and we may be running outdated code.

**Comment by neilkakkar** [comment]
You're right :o - https://github.com/PostHog/posthog/blob/master/requirements.txt#L140 - we should setup the automated bump for this too!

**Comment by samwinslow** [comment]
TODO: bump `posthoganalytics` — may need some verification that there are no breaking changes between 1.1.3 and latest.
